src/MMLib.SwaggerForOcelot/Middleware/SwaggerForOcelotMiddleware.cs

```diff
@@ -59,6 +61,14 @@ public async Task Invoke(HttpContext context)
             var endPoint = GetEndPoint(context.Request.Path);
             var httpClient = _httpClientFactory.CreateClient();
 
+            if (_options.DownstreamSwaggerHeaders != null)
```

Comment:
Please can you extract it into a separate method?

Reply:
Sure.